bin/deploy.sh

```diff
@@ -12,52 +12,37 @@
 #   - Restart OpenAM to take all configuration online
 ####################################################################################
 
-print_help()
+usage() 
 {
-    printf 'Usage: \t%s\t[-c|--cfgdir <cfg directory>] [-n|--namespace <namespace>] [-d|--domain <domain>]
-            \t\t [-p|--prefix <AM prefix>] [-f|--yaml <common yaml file>] [--remove-all] [-h|--help]\n' "$0"
-    printf 'Example 1: %s -p openam -n dev -d forgerock.org\n' "$0"
-    printf 'Example 2: %s -c samples/config/dev\n' "$0"
+    echo "Usage: $0 [-f config.yaml] [-e env.sh] [-R] [-d] config_directory"
```

Comment:
OK, some nice periods at the ends of the second sentences for me :-) lines 17-21